source/simulator/helper_functions.cc

```diff
@@ -461,8 +461,14 @@ namespace aspect
     Utilities::MPI::sum(values, mpi_communicator, values);
 
     const double reference_viscosity = std::exp(values[0]/values[1]);
+    const double length_scale = geometry_model->length_scale();
 
-    return reference_viscosity / geometry_model->length_scale();
+    // Allow the user to inspect and/or overwrite our result:
+    const double result = reference_viscosity / length_scale;
+    if (boost::optional<double> value = signals.modify_pressure_scaling(result, reference_viscosity, length_scale))
+      return *value;
+    else
+      return result;
```

Comment:
You can combine this into
```suggestion
    return signals.modify_pressure_scaling(result, reference_viscosity, length_scale).value_or(result);
```
See https://www.boost.org/doc/libs/1_82_0/libs/optional/doc/html/boost_optional/reference/header__boost_optional_optional_hpp_/detailed_semantics___optional_values.html#reference_optional_value_or_move

Reply:
Whether that's easier to read I don't know.

Reply:
I don't care much either way. Let me know if you want me to make that change (PR is already merged).